### PR TITLE
Added HTTP Proxy info to the collectd page

### PR DESCRIPTION
**Pull request by harnitsignalfx**
@mdubbyap would really appreciate your feedback as well.

### CODE REVIEW

File: collectd/README.md [hunk 1]
```diff
@@ -83,3 +84,23 @@ When the script successfully completes, it starts up collectd, which begins repo
 ## LICENSE
 
 SignalFx version of collectd is released under the Apache 2.0 license. See LICENSE for more details.
+
+## HTTP Proxy
+
+Collectd can talk through an http proxy if needed. The changes would need to be made on files that are sourced by the init scripts.
+
+On CentOS:
+* Modify/Create either of the following files: '/etc/sysconfig/collectd' or '/etc/default/collectd'
```
Comment (mdubbyap) [inline collectd/README.md]: Is this true? Is there a version that one or the other is installed by default?

Comment (harnitsignalfx) [inline collectd/README.md]: This is what I get back from running the command on centos7 > systemctl show collectd

EnvironmentFile=/etc/sysconfig/collectd (ignore_errors=yes)
EnvironmentFile=/etc/default/collectd (ignore_errors=yes)

I'll check on centOS 6 and confirm.

Comment (harnitsignalfx) [inline collectd/README.md]: Ah so init.d/collectd in CentOS 6 only references /etc/sysconfig/collectd

So I'll make it generic to /etc/sysconfig/collectd for both versions of CentOS

Comment (harnitsignalfx) [inline collectd/README.md]: Going to close this one and open a new pull request

**Pull request closed without merging**